core/rest/pom.xml

```diff
@@ -599,73 +523,6 @@
     <!-- While jackson is also a dependency of both jersey and avro, these
          can bring in jars from different, incompatible versions. We force
          the same version with these dependencies -->
```

Comment:
looks good to me Venkat. Should these comments related to jackson be removed too ?